app/presenters/specialist_document_presenter.rb

```diff
@@ -59,7 +59,7 @@ def format_facet_value(type, value, key)
     when "link"
       facet_value_link(key, value)
     when "preset_text"
-      value.map { |v| v[:label] }.join(", ")
+      value.map { |v| v[:label] }
```

Comment:
Is it possible to rewrite this commit message? I'm not sure I fully understand it - are you saying if the type is `link` or `preset_text`, these checks will be moved to other methods in a future commit?

Reply:
updated commit and split